### DISCUSSION TITLE
Disable mobile splash version

### DISCUSSION THREAD

**Discussion by MrHootyHoo** [pH7Software/pH7-Social-Dating-CMS#1074] [Troubleshooting]
I have been looking for hours to find the line that reads the viewport size and determines whether or not to load the "mobile-friendly" Splash so I may disable it. Anyone have any suggestions? 

Also, Thanks for everything, PH!

**Comment by pH-7** [comment]
Great question @MrHootyHoo!

It's done here in the PHP code, `MainController` of the `user` module. https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/2d5a1b9c864328a414b6b924671ba2e6612da96c/_protected/app/system/modules/user/controllers/MainController.php#L51

And then, it will display the splash page if `bIsMobile` is TRUE (when your device is a mobile phone for instance)

```php
} elseif ($this->bIsMobile) { 
     /* 'index.guest.inc.tpl' is not responsive enough for very small screen resolutions, so set to 'index.guest_splash.inc.tpl' by default */ 
     $sPage = static::GUEST_SPLASH_FILE;
}
```
https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/2d5a1b9c864328a414b6b924671ba2e6612da96c/_protected/app/system/modules/user/controllers/MainController.php#L164-L172


Hope it helps @MrHootyHoo! And thanks very much for your kind words 😊 I really appreciate this! 🙂

**Comment by pH-7** [comment]
Finally, if you are interested, this is what `Browser::isMobile()` does

https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/2d5a1b9c864328a414b6b924671ba2e6612da96c/_protected/framework/Navigation/Browser.class.php#L114-L154
